docs/source/developer_guide/development-workflow.md

```diff
@@ -68,6 +68,16 @@ You can check that the JupyterLab extension was successful installed with:
 jupyter labextension list
 ```
 
+> NOTE: 
+When switching between Elyra major versions, it is recommended to clean your JupyterLab environment before a build.
+The `clean-jupyterlab` uninstalls your JupyterLab packages and completely clean/reinstall your Jupyter workspace.
+Make sure to save any important data in your environment before running the script.
+Please refer to the disclaimer at the top of the script file.
```

Comment:
How about echo-ing the disclaimer and forcing the user to confirm when the script is run, like so in `clean-jupyterlab.sh `

```
echo "This script removes your JupyterLab installation ...."
echo "..."
read -p "Hit ENTER to continue or CTRL C to abort"
```

This way all information is in one place.
If there's a use case scenario where the script is part of an automated process (is there?) the last line of code would need to be guarded.

Reply:
I like this approach. 
And afaik it's not automated. @lresende @akchinSTC can confirm.

Reply:
what do you mean by `it's not automated`?

Reply:
for example, the script is not invoked as part of any test or in any way that currently does not require any user interaction to complete